invesalius/gui/widgets/slice_menu.py

```diff
@@ -119,13 +117,11 @@ def __init__(self):
         self.pseudo_color_items[new_id] = color_item
 
         # --------------- Sub menu of the projection type ---------------------
-        self.projection_items = {}
+        self.projection_items: Dict[int, wx.MenuItem] = {}
```

Comment:
Unfortuntately `Dict[wx.WindowIDRef, wx.MenuItem]` results in an error in the line
```python
            self.projection_items[PROJECTIONS_ID[name]] = projection_item
```
since `PROJECTIONS_ID` is a `Dict[str,int]`. The thing is also that type checkers don't understand that `wx.WindowIDRef` is just a wrapper around an int and that its `__hash__` uses the underlying int value so they are exchangeable in indexing
![image](https://github.com/invesalius/invesalius3/assets/113486195/ff591fc1-07d0-4e5d-b566-bded4e2377a7)
I can compromise by making them all a `Dict[int | wx.WindowIDRef, wx.MenuItem]`. What do you think?